SPECS/iptables/iptables

```diff
@@ -43,11 +43,11 @@ echo 0 > /proc/sys/net/ipv4/tcp_ecn
 # Set a known state
 iptables -P INPUT   DROP
 iptables -P FORWARD DROP
-iptables -P OUTPUT  DROP
+iptables -P OUTPUT  ACCEPT
 
 ip6tables -P INPUT   DROP
 ip6tables -P FORWARD DROP
-ip6tables -P OUTPUT  DROP
+ip6tables -P OUTPUT  ACCEPT
```

Comment:
This seems like a big change to the default configuration. Why do we need it? Can this cause any security concerns?

Reply:
This introduces a (brief) period of time where the only rules are to ACCEPT everything. I would say this is a concern